### PR TITLE
184-Release 183 Changes

### PR DESCRIPTION
**Pull request by MattGill98**

### CODE REVIEW

File: check-pages-in-summary.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```
Comment (MarkWareham) [inline check-pages-in-summary.sh]: Is this file supposed to be in the documentation?

Comment (MattGill98) [reply check-pages-in-summary.sh]: No, good spot!

File: release-notes/release-notes-184.adoc [hunk 2]
```diff
@@ -0,0 +1,277 @@
+== New Features
```
[the remark this answers is not in the document]
Comment (MattGill98) [reply release-notes/release-notes-184.adoc]: I thought it had to be there for the sidebar links?

[the remark this answers is not in the document]
Comment (MattGill98) [reply release-notes/release-notes-184.adoc]: Oh I see, I'll remove that!

**Pull request merged**